src/core_plugins/kibana/common/field_formats/types/source.js

```diff
@@ -48,7 +48,7 @@ export function createSourceFormat(FieldFormat) {
   SourceFormat.prototype._convert = {
     text: (value) => toJson(value),
     html: function sourceToHtml(source, field, hit) {
-      if (!field) return this.getConverterFor('text')(source, field, hit);
+      if (!field) return _.escape(this.getConverterFor('text')(source));
```

Comment:
Why aren't `field` and `hit` necessary anymore?

Reply:
No because the `this.getConverterFor` return the previous converter `text: (value) => toJson(value),` that accept only 1 parameter, the text to convert.